cmd/cubic-cli/cmd/transcribe.go

```diff
@@ -395,7 +395,7 @@ func transcribeFiles(workerID int, wg *sync.WaitGroup, client *cubic.Client,
 		case ".mp3":
 			audioEncoding = cubicpb.RecognitionConfig_MP3
 		case ".vox":
-			audioEncoding = cubicpb.RecognitionConfig_VOX8000
+			audioEncoding = cubicpb.RecognitionConfig_ULAW8000
```

Comment:
So, how to we trigger `RecognitionConfig_VOX8000`?

Reply:
We can't yet in this release.  Exxon wants mulaw VOX files and they're currently our only customer who has requested VOX.  Alok's suggestion was that we would have configuration that they can override to specify mulaw for the vox extension, but we haven't implemented that yet.

Reply:
ADPCM may be most common, but mu-law and a-law are also valid vox encodings